test/test_unistd.rs

```diff
@@ -378,6 +379,27 @@ fn test_lseek64() {
     close(tmpfd).unwrap();
 }
 
+// Skip in FreeBSD b/c CI jail environments aren't allowed to change systemwide settings
+#[cfg(not(target_os = "freebsd"))]
+#[test]
+fn test_acct() {
+    skip_if_not_root!("test_acct");
+    let file = NamedTempFile::new().unwrap();
+    let path = file.path().to_str().unwrap();
+
+    acct::enable(path).unwrap();
+    Command::new("echo").arg("Hello world");
+    acct::disable().unwrap();
+
+    let len;
+    loop {
+        len = metadata(path).unwrap().len();
+        if len > 0 { break; }
+    }
+
+    assert!(len > 0);
```

Comment:
The loop makes this assertion guaranteed.  You may as well remove the assertion.

Reply:
Ah I guess I misunderstood the way #[test] blocks of code work. I thought that test cases required an assertion to pass `cargo test`

Reply:
Nope.  They're really quite simple.  Anything that doesn't panic is considered a success.